src/api/app/models/package.rb

```diff
@@ -1323,7 +1323,7 @@ def backend_build_command(command, params)
       # Note: This list needs to keep in sync with the backend code
       permitted_params = params.permit(:repository, :arch, :package, :code, :wipe)
 
-      Suse::Backend.post("/build/#{URI.escape(project.name)}?cmd=#{command}&#{permitted_params.to_h.to_query}", '')
+      Suse::Backend.post("/build/#{URI.escape(project.name)}?cmd=#{command}&#{permitted_params.to_h.to_query}")
```

Comment:
Because of https://github.com/openSUSE/open-build-service/blob/master/src/api/lib/opensuse/backend.rb#L72

Though if that matters here... I dunno